README.md

```diff
@@ -271,6 +287,117 @@ If you access `.stream()` then the byte chunks are provided without storing
 the entire body to memory. Any subsequent calls to `.body()` and `.json()` will
 raise an error.
 
+
+---
+
+## WebSocket
+
+Signature: `WebSocket(scope, recieve, send)`
+
+```python
+from starlette import WebSocket
+
+
+class App:
+    def __init__(self, scope):
+        self.scope = scope
+
+    async def __call__(self, receive, send):
+        ws = WebSocket(self.scope, receive, send)
+
+        # Accept the connection
+        await ws.connect()
+
+        # Recive data
+        name = await ws.receive()
+
+        # Send text
+        await ws.send('hello %s!' % name)
+
+        # Send bytes
+        await ws.send(b'hello world!')
+
+        # Send JSON (as text)
+        await ws.send_json({'data': 'hello world!', 'name': name})
+
+        # Close the socket
+        await ws.close()
+```
```

Comment:
Yup, pretty similar to the API that's in now. Minor differences...

`connect()` is `.accept()`.
`send()` is `.send_text(data)` or `.send_bytes(data)` - I prefer it to note clash with the raw `.send(message)`